src/app/shared/menu/menu-bar.component.css

```diff
@@ -1,8 +1,17 @@
 :host {
-  display: flex;
+  display: block;
   background-color: white;
 }
 
+.menu-bar {
+  display: flex;
+}
+
+.summary {
+  display: flex;
```

Comment:
is this `flex` because of the later stuff we plan to add next to it?

Reply:
yup, yup

Reply:
👍